.circleci/config.yml

```diff
@@ -0,0 +1,186 @@
+version: 2
+
+defaults: &defaults
+  working_directory: /go/src/github.com/weaveworks/scope
+  docker:
+    - image: weaveworks/scope-backend-build:circle-2.0-4ad2f14
+
+client-defaults: &client-defaults
+  working_directory: /home/weave/scope
+  docker:
+    - image: weaveworks/scope-ui-build:circle-2.0-2885645
+
+workflows:
+  version: 2
+  test_and_deploy:
+    jobs:
+      - lint
+      - unit-test
+      - client-build
+      - client-test:
+          requires:
+            - client-build
+      - xplatform-build:
+          requires:
+            - build
+      - build:
+          requires:
+            - client-build
+      - integration-tests:
+          requires:
+            - lint
+            - unit-test
+            - build
+      - deploy:
+          filters:
+            branches:
+              only: master
+          requires:
+            - client-test
+            - integration-tests
+
+jobs:
+  lint:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: make BUILD_IN_CONTAINER=false lint
+
+  unit-test:
+    <<: *defaults
+    parallelism: 1
```

Comment:
Does `parallelism: 1` do anything? As far as I can tell from the docs, I don't think so?

Reply:
No, it's there because I edited it down from 2 to 1 to optimise CircleCI resource usage, and I left the line in place in case I wanted to put it back up again.